### ISSUE TITLE
Zoomable chart with synced axes

### ISSUE THREAD

**Issue by piozygmunt** [amcharts/amcharts5#1654]
Hey. While zooming in on the chart with synced axes, they sometimes loose synchronization. For example in the provided example try to select data from 45 to 49 on axisX. I know syncing axes is kinda expertimental but is there a chance that it will be fixed? Didn't have such problem in amchart4. 

CodePen https://codepen.io/piozygmunt/pen/qBzrvJa

**Environment (if applicable)**
 - amCharts version 5.10.0
 - Browser and its version 128.0.3

**Comment by zeroin** [comment]
I don't think we will fix it. The syncing algorithm is quite the same, most likely it's grid count or size of a chart which might cause the difference.
